components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -388,35 +452,31 @@ public function set_frontier_style_script() {
 			return;
 		}
 
-		// cet the current post type
-		$current_post_type = $this->current_post_type();
+		// cet the current post type.
+		$pod_name = $this->get_pod_name();
 
-		// now use other methods in class to build array to search in/ use
+		// now use other methods in class to build array to search in/ use.
 		$possible_pods = $this->auto_pods();
 
-		if ( isset( $possible_pods[ $current_post_type ] ) ) {
-			$this_pod = $possible_pods[ $current_post_type ];
+		if ( isset( $possible_pods[ $pod_name ] ) ) {
+			$this_pod = $possible_pods[ $pod_name ];
 
-			if ( $this_pod['single'] && is_singular( $current_post_type ) ) {
-				// set template
+			if ( $this_pod['single'] && is_singular( $pod_name ) ) {
 				$template = $this_pod['single'];
 
-			} elseif ( $this_pod['archive'] && is_post_type_archive( $current_post_type ) ) {
-				// if pfat_archive was set try to use that template
-				// check if we are on an archive of the post type
-				// set template
+			} elseif ( $this_pod['archive'] && is_post_type_archive( $pod_name ) ) {
+				// if pfat_archive was set try to use that template.
+				// check if we are on an archive of the post type.
 				$template = $this_pod['archive'];
 
-			} elseif ( is_home() && $this_pod['archive'] && $current_post_type === 'post' ) {
-				// if pfat_archive was set and we're in the blog index, try to append template
-				// set template
+			} elseif ( is_home() && $this_pod['archive'] && $pod_name === 'post' ) {
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 25c48e43f299decc2604a62df91f0ae8</sub>